M2/Macaulay2/d/actors2.dd

```diff
@@ -660,24 +645,6 @@ unSingleton(e:Expr):Expr := (
      else e);
 setupfun("unsequence",unSingleton);
 
-disassemble(e:Expr):Expr := (
-     when e
-     is f:FunctionClosure do toExpr(tostring(Code(f.model)))
-     is f:functionCode do toExpr(tostring(Code(f)))
-     is c:CodeClosure do toExpr(tostring(c.code))
-     is s:SpecialExpr do disassemble(s.e)
-     else WrongArg("pseudocode or a function closure derived from Macaulay 2 code")
-     );
-setupfun("disassemble", disassemble);
```

Comment:
With the new `disassemble` behavior, we've lost some of the previous functionality, like accepting `Function` and `FunctionBody` objects as input.

Reply:
put back those methods